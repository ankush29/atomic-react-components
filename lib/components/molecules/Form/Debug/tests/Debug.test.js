import React from 'react';
import { shallow } from 'enzyme';
import { Debug } from '../Debug';

describe('Debug', () => {
  test('should render correctly', () => {
    expect(shallow(<Debug />)).toMatchSnapshot();
  });
});
