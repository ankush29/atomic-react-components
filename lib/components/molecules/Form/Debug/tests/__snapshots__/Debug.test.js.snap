// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debug should render correctly 1`] = `
<div
  style={
    Object {
      "background": "#f6f8fa",
      "borderRadius": 4,
      "boxShadow": "0 0 1px  #eee inset",
      "margin": "3rem 0",
    }
  }
>
  <div
    style={
      Object {
        "background": "#555",
        "borderTopLeftRadius": 4,
        "borderTopRightRadius": 4,
        "color": "#fff",
        "fontSize": 11,
        "fontWeight": 500,
        "letterSpacing": "1px",
        "padding": ".5rem",
        "textTransform": "uppercase",
      }
    }
  >
    Debug Mode - Form State
  </div>
  <ContextConsumer>
    <Component />
  </ContextConsumer>
</div>
`;
